types/rn-material-ui-textfield/index.d.ts

```diff
@@ -0,0 +1,126 @@
+// Type definitions for rn-material-ui-textfield 1.0
+// Project: https://github.com/gabrieldonadel/rn-material-ui-textfield#readme
+// Definitions by: Craig Duckett <https://github.com/craigwduckett>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import {
+  StyleProp,
+  TextInputProps,
+  TextStyle,
+  ViewStyle,
+  NativeSyntheticEvent,
+  TextInputFocusEventData,
+  TextInputChangeEventData
+} from 'react-native';
+
+export interface ContentInset {
+    top?: number;
+    left?: number;
+    right?: number;
+    label?: number;
+    input?: number;
+}
+
+export interface TextFieldProps extends TextInputProps {
+    textColor?:	string;
```

Comment:
We should use `ColorValue` for colors instead of strings. 
You can import it from react-native like this: 

```js
import { ColorValue } from 'react-native'
```

```suggestion
    textColor?:	ColorValue;
```